src/LibVLCSharp.sln

```diff
@@ -11,7 +11,7 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution
 		global.json = global.json
 	EndProjectSection
 EndProject
-Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Samples", "..\samples", "{799A84A2-2161-4676-878B-5610E3586137}"
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Samples", "Samples", "{799A84A2-2161-4676-878B-5610E3586137}"
```

Comment:
VS generated change? I think you can revert it.

Reply:
Yes, VS is always generate this change, and I think he is right. It's a solution folder, it does not correspond to a physical path.

Reply:
Weird, my VS isn't doing this. No strong feelings.

Reply:
It does it on my side as soon as I make a change to the .sln file (by adding a project, or updating a project build order).